client/my-sites/email/email-management/style.scss

```
@@ -440,3 +440,64 @@
 		}
 	}
 }
+
+.context-all-domain-management {
+	&.main {
+		width: 100%;
+	}
+
+	.navigation-header {
+		border-block-end: none !important;
+	}
+
+	.navigation-header__main {
+		padding-inline: 0 !important;
+
+		.navigation-header__actions {
+			flex-grow: 0;
+		}
+	}
+
+	.email-plan-mailboxes-list__mailbox-list {
+		border: solid 1px var(--color-border-subtle);
+		border-radius: 4px;
+
+		.card {
+			box-shadow: none;
+			background: none;
+		}
+
+		.card.section-header {
+			padding: 1.5rem 0 1rem 0;
+			margin: 0 1.5rem;
+			border-bottom: solid 1px var(--color-border-subtle);
+		}
+
+		.section-header__label-text {
+			display: flex;
+			align-items: center;
+		}
+
+		.section-header__actions {
+			display: flex;
+			flex-direction: row-reverse;
+
+			.components-button.is-link {
+				text-decoration: none;
+			}
+		}
+	}
+
+	.email-plan-mailboxes-list__mailbox-header-icon {
+		display: flex;
+		margin-inline-end: 0.5rem;
+
+		svg {
+			fill: var(--wp-components-color-accent, var(--wp-admin-theme-color, #3858e9));
```

```suggestion
			fill: var(--color-accent);
```
I'm not 100% sure about which variable is the right one to use, but `--wp-components-color-accent` is undefined, and it seems redundant to have a fallback value.

---

True! It's fixed. ✅